### PR TITLE
Update links in sig docs

### PR DESCRIPTION
**Pull request by cblecker**
Does a couple of things:
- Only displays the archive/recording URL if there actually is one in sigs.yaml
- If a label exists for the sig/wg, then display a link to the open PRs/Issues for that sig.

### CODE REVIEW

**Comment by cblecker** [comment]
well that's an unintended consequence with the path labelling.. changing the template (which touches all the sig docs) causes all the sigs to be labelled. I guess at least we know it's working!

File: generator/sig_readme.tmpl [hunk 1]
```diff
@@ -7,8 +7,12 @@
 * [{{.Day}}s at {{.UTC}} UTC]({{$.MeetingURL}}) ({{.Frequency}}). [Convert to your timezone](http://www.thetimezoneconverter.com/?t={{.UTC}}&tz=UTC).
 {{- end }}
 
+{{ if .MeetingArchiveURL -}}
```
Comment (grodrigues3) [inline generator/sig_readme.tmpl]: are we requiring a MeetingArchiveURL (a link to notes) and is there any way to do data validation?  Most sigs seem to have it already? 

This doesn't block this PR or anything.  I'm just curious

Comment (cblecker) [reply generator/sig_readme.tmpl]: I think it's good practice to have it, but I don't think we're imposing it. Yes, all the sigs have it already. If however, for some reason it's missing, this prevents it from displaying a broken link.

File: generator/sig_readme.tmpl [hunk 2]
```diff
@@ -18,6 +22,9 @@ Meeting recordings can be found [here]({{.MeetingRecordingsURL}}).
 ## Contact
 * [Slack](https://kubernetes.slack.com/messages/{{.Contact.Slack}})
 * [Mailing list]({{.Contact.MailingList}})
+{{- if .Label }}
+* [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2F{{.Label}})
```
Comment (xiangpengzhao) [inline generator/sig_readme.tmpl]: sorry for delay. but I think people most possibly want to see a sig's open issues/prs in k/k, not in community. @cblecker WDYT?

Comment (cblecker) [reply generator/sig_readme.tmpl]: Wouldn't be bad to add both. I was more thinking about looking at open design proposals for the sig.

Comment (xiangpengzhao) [reply generator/sig_readme.tmpl]: +1 for adding both :)

**Pull request merged**